examples/custom_role/main.tf

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/******************************************
+  Provider configuration
+ *****************************************/
+provider "google" {
+  version = "~> 3.3"
+}
+
+provider "google-beta" {
+  version = "~> 3.3"
+}
+
+locals {
+  role_permissions = ["iam.roles.list", "iam.roles.create", "iam.roles.delete"]
+}
+
+resource "random_id" "rand_custom_id" {
+  byte_length = 2
+}
+
+/******************************************
+  Module custom_role call
+ *****************************************/
+module "custom-roles" {
```

Comment:
We could update this example to create two custom roles: 1 org-level, 1 project-level.

OR we could just split it into two examples (even better), but 1 fixture invoking both examples.

Reply:
Added an org example. The fixture now calls two examples, one for project and one for org.